From af59a07dad603740d0aaa103cd02b97e4a3267af Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Wed, 1 Apr 2020 11:11:12 +0100
Subject: [PATCH] efi_loader: enable RNG if DM_RNG is enabled

Enable EFI_RNG_PROTOCOL by default if DM_RNG is set.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---

I think it makes sense to enable EFI_RNG_PROTOCOL by default if a config
has enabled both DM_RNG and UEFI.

 lib/efi_loader/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/efi_loader/Kconfig b/lib/efi_loader/Kconfig
index 9890144d41..b9a770281e 100644
--- a/lib/efi_loader/Kconfig
+++ b/lib/efi_loader/Kconfig
@@ -125,6 +125,7 @@ config EFI_GRUB_ARM32_WORKAROUND
 
 config EFI_RNG_PROTOCOL
 	bool "EFI_RNG_PROTOCOL support"
+	default y if DM_RNG
 	depends on DM_RNG
 	help
 	  Provide a EFI_RNG_PROTOCOL implementation using the hardware random
-- 
2.26.0

